### PR TITLE
Add styling section to Collapsible and Accordion

### PR DESCRIPTION
**Pull request by benoitgrelard**
----------

- [x] Change base branch to `main` once #111 is merged. 

----------

### CODE REVIEW

File: pages/primitives/docs/components/accordion/0.0.7/index.mdx [hunk 1]
```diff
@@ -283,6 +283,33 @@ Contains the collapsible content for an item.
   ]}
 />
 
+## Styling
+
+### Animating height
```
Comment (benoitgrelard) [inline pages/primitives/docs/components/accordion/0.0.7/index.mdx:288]: should this be?

```suggestion
### Animating panel height
```

File: pages/primitives/docs/components/collapsible/0.0.7/index.mdx [hunk 2]
```diff
@@ -161,6 +161,33 @@ The component that contains the collapsible content.
   ]}
 />
 
+## Styling
+
+### Animating height
```
Comment (benoitgrelard) [inline pages/primitives/docs/components/collapsible/0.0.7/index.mdx:166]: Should this be?

```suggestion
### Animating content height
```

File: pages/primitives/docs/components/accordion/0.0.7/index.mdx (hunk 1 above)
Comment (jjenzz) [reply pages/primitives/docs/components/accordion/0.0.7/index.mdx:288]: Yeah, I guess so, that is what you're doing afterall

**Review by jjenzz — APPROVED** [review]
I'll leave it up to you if you wanna rename the headings. Makes sense to me but I'm easy

**Pull request merged**